src/stratis_cli/_actions/_logical.py

```diff
@@ -79,8 +79,13 @@ def list_volumes(namespace):
            managed_objects,
            props={'Pool': pool_object_path}
         )
+        fs_names = []
```

Comment:
Given that, as you point out, #101 will make this more like the others, might as well set it up for call to ```print_table``` in the same manner as the others.